server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/XodusServerIssueStore.java

```diff
@@ -37,20 +38,23 @@
 import jetbrains.exodus.entitystore.StoreTransaction;
 import org.jetbrains.annotations.NotNull;
 import org.sonarsource.sonarlint.core.serverconnection.ServerIssue;
+import org.sonarsource.sonarlint.core.serverconnection.ServerTaintIssue;
 
 import static java.util.Objects.requireNonNull;
 
 public class XodusServerIssueStore implements ServerIssueStore {
   private static final String FILE_NAME = "sonarlint.db";
 
   private static final String ISSUE_ENTITY_TYPE = "Issue";
+  private static final String TAINT_ISSUE_ENTITY_TYPE = "Issue";
```

Comment:
Is it normal that we use the same entity type value for both kinds of issues ? If so we could use a single constant, e.g. `ISSUE_COMMON_ENTITY_TYPE`

Reply:
Good catch, it should be a different value

Reply:
Because we have different links so we are able to differentiate them